docs/making-your-own.md

```diff
@@ -24,33 +24,79 @@ However, people like to tinker, so instead of [layering packages](https://docs.f
 
 # Making your Own
 
+## Should I use [`ublue-os/main`](https://github.com/ublue-os/main) or [`ublue-os/startingpoint`](https://github.com/ublue-os/startingpoint)?
+
+The instructions below recommend `startingpoint` because it is:  
+
+- Intended for modification and to be a simple template
+- Better for people making images for personal usage that only build one version
```

Comment:
```suggestion
- Is more suitable for personal usage that only builds one version
```

Reply:
Sorry for linguistic squabble, but I think my version is better because it's not the `personal usage` that `builds`, it's the repo that builds.
Maybe remove the reference to building one version, as startingpoint actually supports that now. I'm off the pc for today so I can't edit.